docs/platforms/native/advanced-usage/signal-handling/index.mdx

```diff
@@ -0,0 +1,62 @@
+* The [glibc-docs](https://www.gnu.org/software/libc/manual/html_node/Signal-Handling.html) and [the man pages](https://man7.org/linux/man-pages/man7/signal.7.html) offer reference details for the signal vocabulary.
+* We recommend chapters 20-22 of Michael Kerrisk's [The Linux Programming Interface](https://man7.org/tlpi/) if you want a book-length treatment.
+ 
+
+## Signals of interest
+
+Signals can support many events, not all of which signify an error or terminate the application (like `SIGCONT`, `SIGUSR1`, or `SIGUSR2`).
+
+In the context of the Native SDK, we are only interested in signals that will result in the application's unexpected termination (crashes). These typically include:
+
+* `SIGSEGV`: the process tried to access an invalid virtual memory "segment".
+* `SIGBUS`: the process tried to access an invalid physical address or a VM page failed to "page-in".
+* `SIGABRT`: raised when `abort()` was called. This is the only signal handled on Windows (and only by the `crashpad` backend) because calling `abort()` bypasses Structured Exception Handling ([`SEH`](https://learn.microsoft.com/en-us/windows/win32/debug/structured-exception-handling)).
+* `SIGFPE`: an erroneous arithmetic operation that occurred in the CPU.
+* `SIGILL`: the process attempted to execute an illegal (often privileged) processor instruction.
+* `SIGTRAP`: typically used to inform debuggers when execution hits a breakpoint instruction (`int 3`, `brk`, etc.), which instrumentation tools and sanitizers sometimes also issue.
+
+If you use the `crashpad` backend, in addition to the above, on Linux, you will also get crash reports for:
+
+* `SIGSYS`: the kernel received a system call with an invalid argument.
+* `SIGEMT`: an emulator trap occurred.
+* `SIGXCPU`: the process exceeded a previously specified CPU [rlimit](https://pubs.opengroup.org/onlinepubs/9699919799/functions/getrlimit.html).
+* `SIGXFSZ`: similar to `SIGXCPU`, raised when a process grows a file beyond a maximum allowed size ([`RLIMIT_FSIZE`](https://man7.org/linux/man-pages/man2/getrlimit.2.html)). 
+
+If your application currently installs handlers for any of the above signals, that can result in a potential conflict. Expressly
+
+* refrain from registering a conflicting signal handler in your application.
+* if you have to, register this handler after `sentry_init()` and let it invoke the previously installed handler (ours) after it finished or you won't get a report.
+* if you or one of your dependencies register a handler for the above signals _before_ you call `sentry_init()`, we will reinstall the previous handler as soon as you call `sentry_close()`. Only the `inproc` backend invokes the previously installed handler at the end of its handler.
+* if you want to act on any crash and not consider specific signals, [register an `on_crash()` hook](https://docs.sentry.io/platforms/native/configuration/filtering/#using-on_crash).
```

Comment:
```suggestion
## Signal Handling Conflicts

Sentry's Native SDK comes with its own set of signal handlers. There may be conflicts if your application currently installs handlers for any [signals of interest](#signals-of-interest) (listed below) that can result in a crash. To avoid this, do the following:

* Refrain from registering a conflicting signal handler in your application.
* If you have to, register this handler after `sentry_init()` and let it invoke the previously installed handler (ours) after it finishes or you won't get a report.
* If you or one of your dependencies register a handler for the signals listed below _before_ you call `sentry_init()`, we will reinstall the previous handler as soon as you call `sentry_close()`. Only the `inproc` backend invokes the previously installed handler at the end of its handler.
* If you want to act on any crash and not consider specific signals, [register an `on_crash()` hook](https://docs.sentry.io/platforms/native/configuration/filtering/#using-on_crash).

### Signals of Interest

In the context of the Native SDK, we are only interested in signals that will result in the application's unexpected termination (crashes). These typically include:

* `SIGSEGV`: The process tried to access an invalid virtual memory "segment".
* `SIGBUS`: The process tried to access an invalid physical address or a VM page failed to "page-in".
* `SIGABRT`: Raised when `abort()` was called. This is the only signal handled on Windows (and only by the `crashpad` backend) because calling `abort()` bypasses Structured Exception Handling ([`SEH`](https://learn.microsoft.com/en-us/windows/win32/debug/structured-exception-handling)).
* `SIGFPE`: An erroneous arithmetic operation that occurred in the CPU.
* `SIGILL`: The process attempted to execute an illegal (often privileged) processor instruction.
* `SIGTRAP`: Typically used to inform debuggers when execution hits a breakpoint instruction (`int 3`, `brk`, etc.), which instrumentation tools and sanitizers sometimes also issue.

If you use the `crashpad` backend, in addition to the above, on Linux, you will also get crash reports for:

* `SIGSYS`: The kernel received a system call with an invalid argument.
* `SIGEMT`: An emulator trap occurred.
* `SIGXCPU`: The process exceeded a previously specified CPU [rlimit](https://pubs.opengroup.org/onlinepubs/9699919799/functions/getrlimit.html).
* `SIGXFSZ`: Similar to `SIGXCPU`, raised when a process grows a file beyond a maximum allowed size ([`RLIMIT_FSIZE`](https://man7.org/linux/man-pages/man2/getrlimit.2.html)). 
```

I think it makes more sense to put the useful info (how to avoid conflicts) up top so it's easier for users to skim. Otherwise, it's easy to miss after a long reference list. I'd also create a new subsection for signals of interest for easy reference.